src/libs/ReportUtils.js

```diff
@@ -1927,7 +1932,7 @@ function buildOptimisticIOUReport(payeeAccountID, payerAccountID, total, chatRep
 function buildOptimisticExpenseReport(chatReportID, policyID, payeeAccountID, total, currency) {
     // The amount for Expense reports are stored as negative value in the database
     const storedTotal = total * -1;
-    const policyName = getPolicyName(allReports[`${ONYXKEYS.COLLECTION.REPORT}${chatReportID}`]);
+    const policyName = Localize.translateIfNeeded(getPolicyName(allReports[`${ONYXKEYS.COLLECTION.REPORT}${chatReportID}`]));
```

Comment:
Why do we need to xrap `getPolicyName` in `Localize.translateIfNeeded`? `getPolicyName` will always return a string, am I missing something?